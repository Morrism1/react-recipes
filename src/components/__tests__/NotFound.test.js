import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import NotFoundPage from '../NotFound';

describe('Component', () => {
  test('should render AC button', () => {
    const { getByText } = render(<NotFoundPage />);
    const link = getByText(/404 Not Found/i);
    expect(link).toBeInTheDocument();
  });

  it('same content with the snapshot', () => {
    const { getByText } = render(<NotFoundPage />);
    const text = getByText('404 Not Found');
    expect(text).toMatchSnapshot();
  });
});
