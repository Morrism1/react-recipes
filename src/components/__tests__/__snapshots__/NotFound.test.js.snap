// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component same content with the snapshot 1`] = `
<h1>
  404 Not Found
</h1>
`;
